src/util/permission/hasPermissionLevelRole.ts

```diff
@@ -0,0 +1,22 @@
+import { GuildMember } from "discord.js";
+import { guildConfigs } from "../../guild/config/guildConfigs";
+import getRoleTypeById from "./getRoleTypeById";
+import { PermissionLevel } from "./permissionLevel";
+import resolvePermissionLevelFromRoleType from "./resolvePermissionLevelFromRoleType";
+
+export function hasPermissionLevelRole(permissionLevel: PermissionLevel, member: GuildMember) {
+    let allRoles: string[] = [];
+	guildConfigs.forEach(
+		g => (allRoles = allRoles.concat(Object.values(g.roles) as string[]))
+	);
+    const roles = member.roles.cache.filter(memberRole =>
```

Comment:
Forgot to include this comment from the copied file
`// We only want to look at roles that we know about from the member`
above L12

No reason to remove it